### ISSUE TITLE
Table class option for dynamic create_table

### ISSUE THREAD

**Issue by pdonorio** [plumdog/flask_table#14]
At the moment create_table works only with the original Table.
If i extend that class, i cannot use it in a dynamic way.

Please provide an additional option to use a different Table class.

Thanks!
Paolo

**Comment by plumdog** [comment]
I've opened a PR for this one too. If you want to try out/review the code from the branch and check it works for you, I'll try to fix/add anything that's missing (same applies to #12). Otherwise I'll just stick a new version on PyPI.

**Comment by pdonorio** [comment]
Thanks, i will look this week into the two PRs.

**Comment by pdonorio** [comment]
Your PR worked for my case:

```python
""" A table test """

from flask_table import Col, create_table, Table

table_name = 'test'
colname = 'coltest'
classes = ['test_one', 'test-hover']
data = [
    {'coltest': 'one'},
    {'coltest': 'three'},
    {'coltest': 'two'},
]


#############################################
class ItemTable(Table):

    def sort_url(self, col_key, reverse=False):
        if reverse:
            direction = 'desc'
        else:
            direction = 'asc'
        return 'test?' + direction


#############################################

TableCls = create_table(table_name, base=ItemTable)
TableCls.add_column(colname, Col(colname))
TableCls.classes = classes
TableCls.allow_sort = True

tmp = TableCls(items=data)
print(tmp.__html__())

# <table class="test_one test-hover">
# <thead><tr><th><a href="test?asc">coltest</a></th></tr></thead>
# <tbody>
# <tr><td>one</td></tr>
# <tr><td>three</td></tr>
# <tr><td>two</td></tr>
# </tbody>
# </table>
```

**Comment by plumdog** [comment]
Merged both and uploaded [new release 0.2.9 to PyPI](https://pypi.python.org/pypi/Flask-Table/0.2.9). Let me know if you run into any problems.
